test/controllers/api/v2/hosts_controller_test.rb

```diff
@@ -314,7 +314,7 @@ def expect_attribute_modifier(modifier_class, args)
 
   test "should update hostgroup_id of host" do
     @host = FactoryBot.create(:host, basic_attrs_with_hg)
-    put :update, params: { :id => @host.to_param, :hostgroup_id => Hostgroup.last.id }
```

Comment:
The last hostgroup doesn't necessarily have an environment, which is required, and reset if the hg changes.